homeassistant/components/homewizard/sensor.py

```diff
@@ -119,6 +119,20 @@
         device_class=SensorDeviceClass.GAS,
         state_class=SensorStateClass.TOTAL_INCREASING,
     ),
+    SensorEntityDescription(
+        key="active_liter_lpm",
+        name="Active Water Usage",
+        native_unit_of_measurement="l/m",
+        icon="mdi:water",
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
+    SensorEntityDescription(
+        key="total_liter_m3",
+        name="Total Water Usage",
+        native_unit_of_measurement=VOLUME_CUBIC_METERS,
+        icon="mdi:gauge",
```

Comment:
There is no device class for something like a water meter or flowsensor. Do we want that?

Reply:
Device classes need an architectural discussion and research on how many integrations would need/support it. If there is a broader use and approval, we could add that (outside of this PR).

Reply:
Thanks for the clarification!